### DISCUSSION TITLE
Disable hardware scroll?

### DISCUSSION THREAD

**Discussion by joey120373** [Bodmer/TFT_eSPI#1568] [General]
I’m working on a project that will need to display both scrolling data ( a serial terminal, like hyper term ) and also display other “pages” that do not scroll. The “terminal” example is pretty much perfect for my needs, though I’ve modified it a bit. The problem I am having is when I want to change to a static screen and just display data, like an analog value. The cursor position does not reset to zero, so the data just appears at whatever point the cursor was set to by the scroll function. I’ve tried a few things, like re-initializing the display, and that works, but takes a while. My next thought was to use the getcursor() function and then just subtract it from itself to get it to zero(x,y) but I haven’t had a chance to try that yet. I am wondering if there is a way to just disable the scroll ? Then re-enable it as needed. This would be a more elegant solution I think. I’ve looked for and info on how to deal with this and I keep coming up with nothing, hoping someone here might have a suggestion. I’m using an ili9341 tft and an RP2040 in 8 bit mode ( performance is very good with this combo! ) . 
thanks

**Comment by Bodmer** [comment]
The simplest way is to leave the scroll area defined but reset the scrollAddress to the correct value, which is the y line coord immediately below the top fixed area. In the TFT_Terminal example the command:
```
yStart = TOP_FIXED_AREA;
scrollAddress(TOP_FIXED_AREA);
```
Will reset the scroll area coordinate datum.

**Comment by joey120373** [comment]
Thank you! I will try that, I think at one point I did try something like

    ystart = 0; 

I didn’t think it would work but I kinda just started messing with anything. 
thanks again

**Reply by joey120373** [reply]
I used your suggestion and it works great! I can now swap back and forth from a scrolling screen to a static screen instantly. Thanks a million !
